### PR TITLE
F/wpc 323 sorting pagination author list

### PR DESCRIPTION
**Pull request by bishoymelek**
Now we have proper pagination working and sorting with the creation date

### CODE REVIEW

**Review by penk85 — DISMISSED** [review]

File: packages/editor/src/client/routes/authorList.tsx [hunk 1]
```diff
@@ -52,7 +71,10 @@ export function AuthorList() {
 
   const authorListQueryVariables = {
     filter: filter || undefined,
-    first: 50
+    skip: page - 1,
+    sort: mapColumFieldToGraphQLField(sortField),
+    first: 50,
```
Comment (roosnic1) [inline packages/editor/src/client/routes/authorList.tsx:76]: Pagination doesn't work if you're always getting 50 articles. I think you have to use `limit` here.

**Review by roosnic1 — CHANGES_REQUESTED** [review]
Looks good. Just one little change

File: packages/editor/src/client/routes/authorList.tsx (hunk 1 above)
Comment (bishoymelek) [reply packages/editor/src/client/routes/authorList.tsx:76]: @roosnic1  Can you explain?

Comment (roosnic1) [reply packages/editor/src/client/routes/authorList.tsx:76]: Add more than 10 authors and test if pagination works with limit set to 10. If you have a limit of 10 you should only see 10 authors.

**Review by roosnic1 — APPROVED** [review]
🎉

**Review by penk85 — APPROVED** [review]

**Pull request merged**